### ISSUE TITLE
[Feature request] Choose the date of the note and color

### ISSUE THREAD

**Issue by logos88** [khuttun/NotificationNotes#18]
Hi! I think this is a really interesting app. What I miss is the option to select a date an a hour if you want the note to pop at a specific time.

On the other hand, it would be nice the option to choose a different color for the notes. Like being able to choose between a few, and use for instance red for things you need to do right now, or in a concrete time, etc.

**Comment by khuttun** [comment]
Setting a note to pop up at specific time starts to get really close to Android's default Calendar app's behavior. As I don't want to duplicate built-in app behavior in Notification Notes, I don't think I will be implementing this.

The coloring of the notes can be considered, though.

**Comment by alanthehat** [comment]
I agree timed notes are covered by the Calendar app, but I think it would be interesting to see if there is a use-case to call into Calendar using 'intents'? Probably only if it's really lightweight though because it would only be a convenience as Calendar is quick & easy to use.
I definitely vote for colours though.

**Comment by NewFoundWorth** [comment]
If you ever add a color option, please consider a possible theme for the entire app, preferably a solid black with white text/buttons. AMOLED screen users who type notes in bed in the dark would greatly appreciate it.  Thanks!
